xOfficeOnlineServer/Misc/Generate_xOfficeOnlineServerInstall.ps1

```diff
@@ -0,0 +1,13 @@
+﻿$oOSInstallIsInstalled = New-xDscResourceProperty -Name Installed -Type Boolean -Attribute Read -Description "Specifies if Office Online Server is installed."

```

Comment:
These files are really used just once to generate the MOF files - do we need to keep them in source control? I feel the MOF file is the important bit here. Plus if people don't use the DscResourceGenerator stuff and just write the MOF by hand we won't have this sort of script. So I would think we could drop this file.